From 52bbd4dd2b644f5469e770df5884e34fbfcf2d8c Mon Sep 17 00:00:00 2001
From: Alastair McKinstry <mckinstry@debian.org>
Date: Mon, 21 Mar 2022 12:34:18 +0200
Subject: [PATCH 8/9] libtool: Change how version is derived

Avoid duplication of '(GNU @PACKAGE@) @VERSION@' in multiple
places by using '$scriptversion'.

* libtoolize.in, build-aux/ltmain.in: Change version from
  '(GNU @PACKAGE@) @VERSION@' to '$scriptversion'.
---
 build-aux/ltmain.in | 2 +-
 libtoolize.in       | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/build-aux/ltmain.in b/build-aux/ltmain.in
index 816b1ca3..c5bdbe5d 100644
--- a/build-aux/ltmain.in
+++ b/build-aux/ltmain.in
@@ -179,7 +179,7 @@ include the following information:
        compiler:       $LTCC
        compiler flags: $LTCFLAGS
        linker:         $LD (gnu? $with_gnu_ld)
-       version:        $progname (GNU @PACKAGE@) @VERSION@
+       version:        $progname $scriptversion
        automake:       `($AUTOMAKE --version) 2>/dev/null |$SED 1q`
        autoconf:       `($AUTOCONF --version) 2>/dev/null |$SED 1q`
 
diff --git a/libtoolize.in b/libtoolize.in
index 5940cc15..6c15aa57 100644
--- a/libtoolize.in
+++ b/libtoolize.in
@@ -140,7 +140,7 @@ When reporting a bug, please describe a test case to reproduce it and
 include the following information:
 
        host-triplet:   @host_triplet@
-       version:        $progname (GNU @PACKAGE@) @VERSION@
+       version:        $progname $scriptversion
        automake:       `($AUTOMAKE --version) 2>/dev/null |$SED 1q`
        autoconf:       `($AUTOCONF --version) 2>/dev/null |$SED 1q`
 
-- 
2.45.2

